opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -313,14 +319,21 @@ impl<R: RuntimeChannel> BatchSpanProcessorInternal<R> {
             let result = export_task.await;
 
             if let Some(channel) = res_channel {
+                // If a response channel is provided, attempt to send the export result through it.
                 if let Err(result) = channel.send(result) {
-                    global::handle_error(TraceError::from(format!(
-                        "failed to send flush result: {:?}",
-                        result
-                    )));
+                    otel_debug!(
+                        name: "BatchSpanProcessor.Flush.SendResultError",
+                        reason = format!("{:?}", result)
+                    );
                 }
             } else if let Err(err) = result {
-                global::handle_error(err);
+                // If no channel is provided and the export operation encountered an error,
+                // log the error directly here.
+                otel_error!(
+                    name: "BatchSpanProcessor.Flush.ExportError",
```

Comment:
unsure why we need Error level internal log for Flush operation? Aren't we returning Flush `Result` back to user already?

Reply:
It doesn't in existing code, can add a TODO. For tracing, the PR is doing search-replace for the global handler.

Reply:
added TODO.